mappings/net/minecraft/SaveVersion.mapping

```diff
@@ -16,8 +16,11 @@ CLASS net/minecraft/class_6595 net/minecraft/SaveVersion
 		COMMENT
 		COMMENT <p>This is stored in the {@code Series} field within {@code level.dat}.
 		COMMENT
-		COMMENT <p>Known values include {@code main} for the main series and
-		COMMENT {@code ccpreview} for the caves and cliffs experimental snapshots.
+		COMMENT <p>Known values include:<ul>
+		COMMENT <li>{@code main} for versions that are not experimental snapshots</li>
+		COMMENT <li>{@code ccpreview} for the 1.18 Caves and Cliffs experimental snapshots</li>
+		COMMENT <li>{@code deep_dark_preview} for 1.19 Deep Dark experimental snapshots</li>
```

Comment:
Snapshots or snapshot?

Reply:
How do we refer to them in fabric or yarn?

Reply:
Not sure, since most of them (I believe) are written before the snapshot phase began. Maybe "snapshots" is fine.